### PR TITLE
[New Algorithm Support] PifPaf

### PR DESCRIPTION
**Pull request by Gyx-One**
Pifpaf Added!
1.Training process
2.Evaluating process

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,12 @@ HyperPose is a library for building human pose estimation systems that can effic
 
 HyperPose has two key features, which are not available in existing libraries:
 
-- **Flexible training platform**: HyperPose provides flexible Python APIs to build many useful pose estimation models (e.g., OpenPose and PoseProposalNetwork). HyperPose users can, for example, customize data augmentation, use parallel GPUs for training, and replace deep neural networks (e.g., changing from ResNet to MobileNet), thus building models specific to their real-world scenarios.
+- **Flexible training platform**: HyperPose provides flexible Python APIs to provide a customise pipeline for developing various pose estimation models. HyperPose users can:
+* make use of uniform pipelines for train,evaluation,visualization,pre-processing and post-processing across various models (e.g., OpenPose,Pifpaf,PoseProposal Network)
+* customise model and dataset for their own use(e.g. user-defined model,user-defined dataset,mitiple dataset combination)
+* 
+* parallel training using multiple GPUs(using *Kungfu* adaptive distribute training library)
+thus building models specific to their real-world scenarios.
```
Comment (ganler) [inline README.md]:
```suggestion
**Flexible training platform**: HyperPose provides flexible Python APIs to build many useful pose estimation models (e.g., OpenPose and PoseProposalNetwork). HyperPose users can, for example, customize data augmentation, use parallel GPUs for training, and replace deep neural networks (e.g., changing from ResNet to MobileNet), thus building models specific to their real-world scenarios.
```

**Review by ganler — COMMENTED** [review]
Hi, Yixiao, this is excellent. By the way, please take a look at https://github.com/tensorlayer/hyperpose/pull/320, to see if our python training codes are compatible with tensorflow 2.1.2. 
Besides, you may format our python codes using some tools(e.g., flake8).

**Comment by Gyx-One** [comment]
> Hi, Yixiao, this is excellent. By the way, please take a look at #320, to see if our python training codes are compatible with tensorflow 2.1.2.
> Besides, you may format our python codes using some tools(e.g., flake8).

Ok, thanks, I will have a check

**Comment by Gyx-One** [comment]
Already checked, tensorflow=2.1.2 is compatible.

**Review by zsdonghao — APPROVED** [review]

**Pull request merged**